app/views/individual/incometax/business/OverseasPropertyAccountingMethod.scala.html

```diff
@@ -0,0 +1,87 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.data.Form
+@import uk.gov.hmrc.play.views.html.helpers.form
+@import views.html.helpers._
+@import config.AppConfig
+@import forms.individual.business.AccountingMethodOverseasPropertyForm
+@import forms.submapping.AccountingMethodMapping
+@import models.common.OverseasAccountingMethodPropertyModel
+@import views.html.templates.PrincipalMainTemplate
+@import views.individual.helpers._
+@import views.html.helpers.injected.{ContinueButton, RadioHelper, DetailsHelper}
+
+@this(
+    mainTemplate: PrincipalMainTemplate,
+    form: FormWithCSRF,
+    detailsHelper: DetailsHelper,
+    radioHelper: RadioHelper,
+    continueButton: ContinueButton
+)
+
+@(overseasPropertyAccountingMethodForm: Form[OverseasAccountingMethodPropertyModel], postAction: Call, isEditMode: Boolean, backUrl: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@question = @{
+    messages("overseas.property.accounting_method.heading")
+}
+
+@detailsContent = {
+    <p>@Messages("property.accounting_method.accordion.content.para")</p>
+    <ul class="bullets">
```

Comment:
```suggestion
    <ul class="govuk-list govuk-list--bullet">
```
wrong class, should be as suggested. I also saw that the same thing was done on the uk property accounting method page, can that be fixed?